docs/modules/ROOT/pages/documentation/ecosystem/payara-jax-rs-ssl.adoc

```diff
@@ -0,0 +1,36 @@
+= Payara JAX-RS Client Automatic SSL Context
+
+JAX-RS Client Automatic SSL Context is an implementation to automatically indicate the certificate when doing a Rest Client call
```

Comment:
This sentence is a bit wordy, and I'm not sure it actually describes what the feature is doing - it's not automatic.